.github/workflows/release_mac.yml

```diff
@@ -64,10 +64,9 @@ jobs:
         source workflow_scripts/protobuf/build_protobuf_unix.sh $NUM_CORES $(pwd)/protobuf/protobuf_install
         if [ '${{ github.event_name }}' == 'schedule' ]; then
           sed -i 's/name = "onnx"/name = "onnx-weekly"/' 'pyproject.toml'
-          ONNX_PREVIEW_BUILD=1 python -m build --wheel
-        else
-          python -m build --wheel
+          export ONNX_PREVIEW_BUILD=1
```

Comment:
Didn't know this will not work. Do you know why?

Reply:
IIRC, on Mac `ONNX_PREVIEW_BUILD=1 python -m build --wheel` won't use `ONNX_PREVIEW_BUILD=1` for that command.